.github/workflows/blank.yml

```diff
@@ -35,3 +35,22 @@ jobs:
     # Run the test cases
     - name: Run the test cases
       run: docker run --volume $(pwd)/tests:/tests tools_cloudshell pwsh -c "cd /tests; Install-Module -Name Pester -Force; Invoke-Pester -EnableExit"
+    
+    # Show Docker image size
+    - name: find the pull request id
+      run: echo ::set-env name=ISSUEID::$(echo "${{github.ref }}" | sed 's!refs/pull/\([0-9]*\)/merge!\1!') 
+    - name: find the base size info
+      run: echo ::set-env name=BASE_SIZE::$(docker inspect base_cloudshell:latest --format "{{.Size}}")
+    - name: find the tools size info
+      run: echo ::set-env name=TOOLS_SIZE::$(docker inspect tools_cloudshell:latest --format "{{.Size}}")
+    - name: update a comment with size
+      run: |
+        echo "pull id $ISSUEID size $BASE_SIZE  $TOOLS_SIZE"  && \
+        curl --request POST \
+        --url https://api.github.com/repos/${{ github.repository }}/issues/$ISSUEID/comments \
+        --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+        --header 'content-type: application/json' \
+        --header 'Accept: application/vnd.github.v3+json' \
+        --data "{
+          \"body\": \"Image size with this change is base: $(($BASE_SIZE / 1048576))MB, tools: $(($TOOLS_SIZE / 1048576))MB. \"
```

Comment:
It will be nice if image size change is also shown here, e.g. this change introduced 200KB+ in base image, 500KB+ in tools image.

Reply:
I thought about it. Not impossible but makes its bit more complicated. 

1) where do we save the size to compare to? 
2) what do we compare to - what's in master? Most recent release?

Think I will merge this for now and figure out deltas another time.